app/views/layouts/_flash.html.erb

```diff
@@ -2,7 +2,11 @@
   <% flash.each do |type, msg| %>
     <div class="alert <%= bootstrap_class_for_flash(type) %> alert-dismissable fade show">
       <%= msg %>
-      <button class="close" data-dismiss="alert">x</button>
+      <% if flash_message_is_cookie_info(msg) && cookies[:saw_cookie_notice] == nil %>
+        <%= link_to('X', saw_cookie_notice_path, method: :post, class: 'float-right') %>
+      <% else %>
+        <button class="close" data-dismiss="alert">x</button>
+      <% end %>
```

Comment:
When both an alert and a cookie flash is present, the "close" button won't be displayed. We need to display both the cookie info + alert message under each other.

Reply:
@oborze I think you're wrong here, please check once again. I have cookie alert and ie. info about sign in in two separate boxes, both of them have close button.

Reply:
Yup, sorry!